packaging/Makefile

```diff
@@ -120,13 +120,24 @@ build/windows/amd64/%:
 	GOOS=windows \
 	GOARCH=amd64 \
 	CGO_ENABLED=$(CGO_ENABLED) \
-	go build -ldflags "-X main.version=$(VERSION) -X main.commit=$(COMMIT_HASH)" -o build/windows/amd64/$(notdir $@) ../cmd/$(notdir $@)/*.go
+	go build -ldflags "-X main.version=$(VERSION) -X main.commit=$(COMMIT_HASH)" -o build/windows/amd64/$(notdir $@).exe ../cmd/$(notdir $@)/*.go
 
 build.package: build.linux build.darwin build.windows
+# Pack linux binaries as tar
 	tar --transform 's,^\.,skipper-$(VERSION)-linux-amd64,' -C build/linux/amd64 -czvf skipper-$(VERSION)-linux-amd64.tar.gz .
 	tar --transform 's,^\.,skipper-$(VERSION)-linux-arm64,' -C build/linux/arm64 -czvf skipper-$(VERSION)-linux-arm64.tar.gz .
 	tar --transform 's,^\.,skipper-$(VERSION)-linux-armv7,' -C build/linux/arm/v7 -czvf skipper-$(VERSION)-linux-armv7.tar.gz .
+
+# Pack darwin binaries as tar
 	tar --transform 's,^\.,skipper-$(VERSION)-darwin-amd64,' -C build/darwin/amd64 -czvf skipper-$(VERSION)-darwin-amd64.tar.gz .
 	tar --transform 's,^\.,skipper-$(VERSION)-darwin-arm64,' -C build/darwin/arm64 -czvf skipper-$(VERSION)-darwin-arm64.tar.gz .
-	tar --transform 's,^\.,skipper-$(VERSION)-windows-amd64,' -C build/windows/amd64 -czvf skipper-$(VERSION)-windows-amd64.tar.gz .
+
+# Pack windows binaries as zip - use a temp directory to achieve same transform result as other archives
+	mkdir skipper-$(VERSION)-windows-amd64
+	for f in ./build/windows/amd64/*.exe; do cp "$$f" "skipper-$(VERSION)-windows-amd64"; done
+	zip -r skipper-$(VERSION)-windows-amd64.zip skipper-$(VERSION)-windows-amd64
+	rm -rf skipper-$(VERSION)-windows-amd64
+
+# Generate checksums for all archives and save to file
 	for f in *.tar.gz; do sha256sum $$f >> sha256sum.txt; done
+	for f in *.zip; do sha256sum $$f >> sha256sum.txt; done
```

Comment:
I think this could be simplified a bit:
```diff
diff --git a/packaging/Makefile b/packaging/Makefile
index ddfe36c9..46254501 100644
--- a/packaging/Makefile
+++ b/packaging/Makefile
@@ -132,12 +132,10 @@ build.package: build.linux build.darwin build.windows
        tar --transform 's,^\.,skipper-$(VERSION)-darwin-amd64,' -C build/darwin/amd64 -czvf skipper-$(VERSION)-darwin-amd64.tar.gz .
        tar --transform 's,^\.,skipper-$(VERSION)-darwin-arm64,' -C build/darwin/arm64 -czvf skipper-$(VERSION)-darwin-arm64.tar.gz .
 
-# Pack windows binaries as zip - use a temp directory to achieve same transform result as other archives
-       mkdir skipper-$(VERSION)-windows-amd64
-       for f in ./build/windows/amd64/*.exe; do cp "$$f" "skipper-$(VERSION)-windows-amd64"; done
+# Pack windows binaries as zip
+       cp -R build/windows/amd64/ skipper-$(VERSION)-windows-amd64
        zip -r skipper-$(VERSION)-windows-amd64.zip skipper-$(VERSION)-windows-amd64
        rm -rf skipper-$(VERSION)-windows-amd64
 
 # Generate checksums for all archives and save to file
-       for f in *.tar.gz; do sha256sum $$f >> sha256sum.txt; done
-       for f in *.zip; do sha256sum $$f >> sha256sum.txt; done
+       sha256sum *.tar.gz *.zip
```

Reply:
Ah, of course! Invaluable feedback. I have fixed everything above, and tested it. Still works.